e2e_samples/parking_sensors/scripts/deploy_azdo_variables.sh

```diff
@@ -91,55 +91,57 @@ az pipelines variable-group create \
         databricksNotebookPath="$databricksNotebookPath" \
         databricksClusterId="$databricksClusterId" \
         apiBaseUrl="$apiBaseUrl" \
-    --output json
+     -o none
```

Comment:
I know you tested it - but I'm just not sure if the indentation here is right? Can you just confirm?